skycultures/indian/index.json

```diff
@@ -737,49 +737,49 @@
-			 { "symbol":  "5", "native": "मृगशिर",            "pronounce": "Mrigashira", "english": "Deer's Head"},
-			 { "symbol":  "6", "native": "आर्द्रा",             "pronounce": "Ardra",      "english": "Storm God"},
-			 { "symbol":  "7", "native": "पुनर्वसु ",            "pronounce": "Punarvasu",  "english": "Two Restorers of Goods"},
-			 { "symbol":  "8", "native": "पुष्य",              "pronounce": "Pushya",    "english": "Nourisher"},
-			 { "symbol":  "9", "native": "आश्ळेषा/आश्लेषा",       "pronounce": "Āshleshā",   "english": "Embrace"},
-			 { "symbol": "10", "native": "मघा",              "pronounce": "Maghā",     "english": "Bountiful"},
-			 { "symbol": "11", "native": "पूर्व फाल्गुनी",          "pronounce": "Pūrva Phalgunī", "english": "First Reddish One"},
-			 { "symbol": "12", "native": "उत्तर फाल्गुनी",         "pronounce": "Uttara Phalgunī", "english": "Second Reddish One"},
-			 { "symbol": "13", "native": "हस्त",              "pronounce": "Hasta",     "english": "Hand"},
-			 { "symbol": "14", "native": "चित्रा",              "pronounce": "Chitra",    "english": "Bright One"},
-			 { "symbol": "15", "native": "स्वाति",             "pronounce": "Svati",     "english": "Very good"},
-			 { "symbol": "16", "native": "विशाखा",            "pronounce": "Vishakha",   "english": "Forked, having branches"},
-			 { "symbol": "17", "native": "अनुराधा",           "pronounce": "Anuradha",    "english": "Following rādhā"},
-			 { "symbol": "18", "native": "ज्येष्ठा",             "pronounce": "Jyeshtha",   "english": "Eldest, most excellent"},
-			 { "symbol": "19", "native": "मूल",              "pronounce": "Mula",       "english": "Root"},
-			 { "symbol": "20", "native": "पूर्व आषाढा",          "pronounce": "Purva Ashadha",  "english": "First Invincible"},
-			 { "symbol": "21", "native": "उत्तर आषाढा",         "pronounce": "Uttara Ashadha", "english": "Later Invincible"},
-			 { "symbol": "22", "native": "श्रवण",             "pronounce": "Shravana",       "english": ""},
-			 { "symbol": "23", "native": "श्रविष्ठा/धनिष्ठा",         "pronounce": "Dhanishta/Shravishthā", "english": "Most Famous/Swiftest"},
-			 { "symbol": "24", "native": "शतभिष/शततारका",      "pronounce": "Shatabhisha Nakshatra", "english": "Requiring a hundred Physicians"},
-			 { "symbol": "25", "native": "पूर्व भाद्रपदा/पूर्व प्रोष्ठपदा",   "pronounce": "Purva Bhadrapada",  "english": "First of the Blessed Feet"},
-			 { "symbol": "26", "native": "उत्तर भाद्रपदा/उत्तर प्रोष्ठपदा", "pronounce": "Uttara Bhādrapadā", "english": "Second of the Blessed Feet"},
-			 { "symbol": "27", "native": "रेवती",            "pronounce": "Revati",            "english": "Prosperous"}, 
-			 { "symbol": "28", "native": "अभिजित",          "pronounce": "Abhijit",           "english": "Invincible"}]},			 
+	"names": [ { "symbol":  "Aś", "native": "अश्विनी",             "pronounce": "Ashvini",    "english": "Physician to the Gods"},
```

Comment:
likewise here, native should be Sanskrit, pronounce should be Latin-letter transcription with diacritics (and maybe decide on which word exactly when two are given like "Dhanishta/Shravishthā"), and "english" should be a translation. The symbol is plotted in the lower right corner of each nakshatra, and is never used as alternative "short" label, therefore I think showing the numerical sequence is better.

Reply:
> native should be Sanskrit

Sanskrit is a language, written in a variety of scripts (though currently dominated by devanAgarI); including latin. So, technically, it should be fine since sanskrit speakers from non-devanAgarI regions too might find it easier.

>  I think showing the numerical sequence is better.

If you ask my what's naxatra 13 is, I couldn't say without reciting the previous 12 from memory. So, I felt the symbol should be something more meaningful.

Reply:
> So, technically, it should be fine since sanskrit speakers from non-devanAgarI regions too might find it easier.

Well, the dominant script should be considered "native" I suppose, and those who will find the latinized version easier can choose the option to show it instead if it's put in "pronounce" (in the current development version of Stellarium).

Reply:
Done - nice to see native script indeed!

![image](https://github.com/user-attachments/assets/0d664e5a-e514-412f-90d5-7d170edc2e6f)